src/Database/Esqueleto/Internal/Internal.hs

```diff
@@ -1983,10 +2078,24 @@ instance Semigroup LimitClause where
 
 instance Monoid LimitClause where
   mempty = Limit mzero mzero
-  mappend = (<>)
 
 -- | A locking clause.
-type LockingClause = Monoid.Last LockingKind
+data LockingClause =
+    LegacyLockingClause LockingKind
+    -- ^ Locking clause not specific to any database implementation
+    | PostgresLockingClauses [PostgresLockingKind]
+    -- ^ Locking clause specific to postgres
+    | NoLockingClause
```

Comment:
Can we put `@since` annotations on these?

Reply:
I believe `LockingClause` is internal only - do you still want me to add?